packages/blueprints/test-blueprint/src/defaults.json

```diff
@@ -1,4 +1,34 @@
 {
+  "selectors": {
+    "sourceRepo": "repo-name-does-not-exist",
+    "sourceRepoOrAdd": "existing-repo",
+    "blueprintInstantiation": "prefilled-id-that-does-not-exist"
+  },
+  "multiSelectors": {
+    "stringMulti": ["C", "B"],
+    "numberMulti": [100]
+  },
+  "toupleValues": {
+    "singles": {
+      "doubleTouple": ["hello", "moto"],
+      "doubleToupleNum": [123, 456]
+    },
+    "lists": {
+      "doubleTouple": [
+        ["team", "first"],
+        ["team", "second"]
+      ],
+      "doubleToupleNum": [
```

Comment:
these default values violate the specified regex since there's a space in first team, second team, and third team